examples/04_fuzzy_joining.py

```diff
@@ -119,15 +119,30 @@
 # To join them with skrub, we only need to do the following:
 from skrub import fuzzy_join
 
-df1 = fuzzy_join(
+augmented_df = fuzzy_join(
     df,  # our table to join
-    gdppc,  # the table to join with
+    gdp_per_capita,  # the table to join with
     left_on="Country",  # the first join key column
     right_on="Country Name",  # the second join key column
-    return_score=True,
+    insert_match_info=True,
 )
 
-df1.tail(20)
+augmented_df.tail(20)
+
+###############################################################################
+# In this case, it is better to use the threshold parameter
+# so as to include only precise-enough matches:
+#
+augmented_df = fuzzy_join(
+    df,
+    gdp_per_capita,
+    left_on="Country",
+    right_on="Country Name",
+    max_dist=0.9,
+    insert_match_info=True,
+)
+augmented_df.sort_values("skrub_Joiner_rescaled_distance", ascending=False).head()
+
 # We merged the first World Bank table to our initial one.
 
 ###############################################################################
```

Comment:
I can't comment on it but L151 here you have a note saying 
``We fix the return_score parameter to True so as to keep the matching score, that we will use later to show what are the worst matches.
``
It might go like this:
``We fix the insert_match_info parameter to True so as to keep the string distances, that we use to show what are the worst matches.
``